site/docs/master/supported-providers.md

```diff
@@ -0,0 +1,77 @@
+# Providers
+
+Velero can store its backups in any S3 or S3 compatible object storage.
+
+## S3-Compatible object store providers
+
+Velero uses [Amazon's Go SDK][0] to connect to the AWS S3 API. Some third-party storage providers also support the S3 API, and users have reported the following providers work with Velero:
```

Comment:
We might want to reword this to say "Velero's AWS Object Store plugin uses Amazon's Go SDK..."